### PR TITLE
add overflow-y: hidden to mathjax containers

### LINKED ISSUE #674: Add `overflow-y: hidden;` to css style `mjx-container`
#### Problem Description

A clear and concise description of the problem you are trying to solve.

Currently `math.html.jinja2` sets `overflow-x: auto;` [see this permalink](https://github.com/mitmproxy/pdoc/blob/7b0745211f701f0ea6e9bc94a62c3d2c0c15a5eb/pdoc/templates/math.html.jinja2#L26C9-L26C26)

In my version of Chrome/Edge has the unintended(?) effect of causing unnecessary scroll bars to appear on the vertical / y axis. To counteract this behaviour we can set `overflow-y: hidden;`

Note that this change should not actually hide anything, since the height of the container is already correctly set by mathjax.

#### Proposal
Change the style in [math.html.jinja2](https://github.com/mitmproxy/pdoc/blob/7b0745211f701f0ea6e9bc94a62c3d2c0c15a5eb/pdoc/templates/math.html.jinja2#L26C9-L26C26) to below

```css
mjx-container {
    overflow-x: auto;
    overflow-y: hidden;
}
```

#### Alternatives

Not sure

#### Additional context

Before the proposed change

image

After the proposed change

image

### PR DESCRIPTION
**Pull request by thehappycheese**
closes #674 

Adds `overflow-y:hidden` to the mathjax container template CSS.

This removes unnesisary vertical scroll bars that show up in Chrome and Edge when `overflow-x` is set to `auto`, particularly for multi-line display mode math blocks.

<details>

<summary>Click Here to expand source code for images below</summary>

`test_module.py`:
```python
r"""
# Inline math tests

this is an $i=n$ line math expression

# Single line display mode

Long (always needs horizontal scroll)

$$
5a^3b^2 - 2b^4c + 7c^2d^3e - 3d^2ef^2 + 6e^3f - 4a^2f^3 + 9a^4bc^2 - 8b^3d^2e^2 + 3c^5de - 7d^3e^4f + 2a^2b^2c^3 - 6b^5df^2 + 4c^3d^4e^2 - 9d^2e^3f^3 + 5a^3cf^4 - 2b^2d^5e + 8c^4d^2f - 6a^5be^3 + 7b^3c^2d^3
$$

Short (never needs scroll)

$$
5a^3b^2 - 2b^4c + 7c^2d^3e - 3d^2ef^2
$$

# Multi line display mode

$$
\\begin{align}
2x^2 + 3y - 7z &= 5 \\\\
11 &= 4x - 6y^3 + 2z^2 + 5a^3b^2 - 2b^4c + 7c^2d^3e - 3d^2ef^2 + 6e^3f - 4a^2f^3 \\\\
9x^3 - 5y^2 + z &= 3
\\end{align}
$$

"""
```

</details>

### Before

image

### After

image

### CODE REVIEW

**Review by mhils — APPROVED** [review]
Thanks! :cake:
